yodeploy/cmds/yodeploy_server.py

```diff
@@ -29,11 +30,42 @@ def parse_args(flask_app):
     return opts
 
 
+def create_context():
+    if hasattr(ssl, 'create_default_context'):  # Python >= 2.7.9
+        context = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
+        context.options |= ssl.OP_CIPHER_SERVER_PREFERENCE
```

Comment:
without ever having set up an SSL context before, this line of code is greek to me and your commit message is not illuminating

Reply:
Fair point. SSL Handshake starts with a client sending a ClientHello packet, containing (amongst other things) an ordered list of ciphers that the client can use. By default, OpenSSL servers use the first of these that the server supports. If you set this flag, the server selects the first certificate from its own list, that the client supports. So the server can override the client's preferences.